README.md

```diff
@@ -32,7 +32,9 @@ To get a local copy up and running follow these simple example steps.
 
 ### Usage
 
-### Run tests
+### How to Run
+
+#!/usr/bin/env ruby
 
 ### Deployment
```

Comment:
- [ ] Please update your README file by removing some sections you do not need also kindly add the setup, install and how to run the project locally

![image](https://user-images.githubusercontent.com/75126481/174290759-b43b3fa6-a588-43f2-9c13-4389d31ed7a8.png)